test-data/unit/check-protocols.test

```diff
@@ -2427,7 +2427,7 @@ from typing import Protocol
 class P(Protocol): ...
 class C(P): ...
 
-reveal_type(C.register(int))  # E: Revealed type is 'def () -> builtins.int'
+reveal_type(C.register(int))  # E: Revealed type is 'def (x: builtins.object =, base: builtins.int =) -> builtins.int'
```

Comment:
Not sure why the revealed type is the constructor signature -- when I try directly running mypy on this, I get `Type[builtins.int]` instead. I guess this is just some test-related artifact?